apps/framework-cli/src/framework/python/wrappers/scripts/python_worker_wrapper/activity.py

```
@@ -2,16 +2,64 @@
 from dataclasses import dataclass
 from moose_lib.dmv2 import get_workflow
 from moose_lib.dmv2.workflow import TaskContext
-from typing import Optional, Callable
+from typing import Optional, Callable, Any, TextIO
```

Ruff (0.14.11)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)